stable/aws-for-fluent-bit/values.yaml

```diff
@@ -43,6 +43,7 @@ filter:
   mergeLogKey: "data"
   k8sLoggingParser: "On"
   k8sLoggingExclude: "On"
+  bufferSize: "32k"
```

Comment:
Just noting that 32KB is the current default

Reply:
@PettitWesley 
Thanks for your comment. The following document says 32k, is that wrong?
https://docs.fluentbit.io/manual/pipeline/filters/kubernetes

Reply:
No it isn't wrong... I think I might have made this comment since I wasn't sure if we should just copy the same default or create a different one here